### PR TITLE
First review

### PR DESCRIPTION
**Pull request by lorihughes**

### CODE REVIEW

File: src/D2662.tex [hunk 1]
```diff
@@ -19,108 +19,117 @@
...
-More importantly, it may close the door to future evolutions.
-\paper{P1858R0} proposes \tcode{tuple-like.[N]} as sugar over \tcode{get<N>(tuple-like)}
-and \tcode{aggregate.[N]} returning the $Nth$ data member of an aggregate.
+\paper{P1858R0} proposes \tcode{tuple-like.[N]} as sugar over % lah: What do you mean "as sugar over"? If I'm not clear, then a global audience won't be clear. 
+\tcode{get<N>(tuple-like)}
+and \tcode{aggregate.[N]} returning the $Nth$ data member of an aggregate. % lah: WHAT is "returning the $Nth$ data member of an aggregate"---just the aggregate.[N] or BOTH get<N>(tuple-like) AND aggregate.[N]? 
 
-Ideally, \tcode{tuple[N]} would just work. A few proposals have tried to improve the ergonomy of tuple indexing
-\paper{P2726R0}, \paper{P0311R0}.
+Ideally, \tcode{tuple[N]} would just work. % lah: By "just," do you mean that it would simply work or do you mean that it would barely work?
```
Comment (cor3ntin) [inline src/D2662.tex:131]: The former

File: src/D2662.tex [hunk 2]
```diff
@@ -19,108 +19,117 @@
...
-\paper{P1858R0} proposes \tcode{tuple-like.[N]} as sugar over \tcode{get<N>(tuple-like)}
-and \tcode{aggregate.[N]} returning the $Nth$ data member of an aggregate.
+\paper{P1858R0} proposes \tcode{tuple-like.[N]} as sugar over % lah: What do you mean "as sugar over"? If I'm not clear, then a global audience won't be clear. 
+\tcode{get<N>(tuple-like)}
+and \tcode{aggregate.[N]} returning the $Nth$ data member of an aggregate. % lah: WHAT is "returning the $Nth$ data member of an aggregate"---just the aggregate.[N] or BOTH get<N>(tuple-like) AND aggregate.[N]? 
 
-Ideally, \tcode{tuple[N]} would just work. A few proposals have tried to improve the ergonomy of tuple indexing
-\paper{P2726R0}, \paper{P0311R0}.
+Ideally, \tcode{tuple[N]} would just work. % lah: By "just," do you mean that it would simply work or do you mean that it would barely work? 
+A few proposals have tried to improve the ergonomy % lah: Do you mean ergodicity?
```
Comment (cor3ntin) [inline src/D2662.tex:132]: user-friendliness is probably what we want to say.

File: src/D2662.tex [hunk 3]
```diff
@@ -19,108 +19,117 @@
...
 \subsubsection{pack.[index]}
 
 Historically in C++ (and C++-like languages), a single dot denotes member access.
-Reusing that syntax for pack indexing would introduce a semantics inconsistency.
+Reusing that syntax for pack indexing would introduce a semantics inconsistency and, more importantly, could close the door to future evolutions.
 
-More importantly, it may close the door to future evolutions.
-\paper{P1858R0} proposes \tcode{tuple-like.[N]} as sugar over \tcode{get<N>(tuple-like)}
-and \tcode{aggregate.[N]} returning the $Nth$ data member of an aggregate.
+\paper{P1858R0} proposes \tcode{tuple-like.[N]} as sugar over % lah: What do you mean "as sugar over"? If I'm not clear, then a global audience won't be clear.
```
Comment (cor3ntin) [inline src/D2662.tex:127]: "syntactic sugar"

File: src/D2662.tex [hunk 4]
```diff
@@ -19,108 +19,117 @@
...
 \item We want a syntax that can be expanded to support slicing in the future.
-\item Pack elements can be array-like or tuple-like, so we need to be careful
-  about ambiguities.In particular, directly applying a subscript to a pack
-  (\tcode{P[i]}, where \tcode{P} is a pack) is not viable. Indeed, indexing a
+\item Pack elements can be array like or tuple like, so we need to be careful
+  about ambiguities. In particular, directly applying a subscript to a pack
+  (\tcode{P[i]}, where \tcode{P} is a pack) is nonviable. Indeed, indexing a
   pack of arrays (\tcode{ArrayPack...[index]}), indexing each array in a pack
-  (\tcode{ArrayPack[index]...}), indexing a single array with a pack of indexes
+  (\tcode{ArrayPack[index]...}), indexing a single array with a pack of indices % lah: indexes is for book indexes and indices is for mathematics.
```
Comment (cor3ntin) [inline src/D2662.tex:111]: Alas, I think C++ uses "index" for these things so I'm not sure "indices" would be more correct in the context of what c++ committee members expect

File: src/D2662.tex [hunk 5]
```diff
@@ -19,108 +19,117 @@
...
-
-\subsection{Other syntaxes considered}
+This syntax is used by Circle and was initially proposed by \paper{P1858R2}.
 
-The pack...[index] syntax was selected for this proposal after considering a
-number of other possible syntaxes, some of which have been proposed by other
-committee members:
+\subsection{Other syntactic options considered}
 
+The pack...[index] % lah: Should this be in code font?
```
Comment (cor3ntin) [inline src/D2662.tex:77]: Yes!

File: src/D2662.tex [hunk 6]
```diff
@@ -19,108 +19,117 @@
...
+\begin{itemize}
+\item{Initial revision}
+\end{itemize}
 
 \section{Motivation}
 
-The motivation for pack indexing is covered in "Generalized pack declaration and usage" (\paper{P1858R2}) and
-"A plan for better template meta programming facilities in C++26" \cite{P2632R0}.
+The motivation for pack indexing is covered in \paper{P1858R2} and
+ \cite{P2632R0}. % lah: 1. Why are you using two different citation methods? Both papers are cited in this paper. When I click the first paper link, I'm linked to the paper online. When I click the second paper link, I'm linked to the item in the bib. Why are these different? 2. You don't list titles for citations elsewhere, so it's inconsistent to list them here.
```
Comment (cor3ntin) [inline src/D2662.tex:45]: it should be \paper everywhere, I'll fix that!

File: src/D2662.tex [hunk 7]
```diff
@@ -19,108 +19,117 @@
 
 \section{Abstract}
 
-This paper expands on the pack indexing feature described in \paper{P1858R2} and provides wording.
+This paper expands on the pack indexing feature described in \paper{P1858R2} and provides wording. % lah: wording for what? Perhaps add something like "proposes wording changes for the Standard." 
+
+% lah: do you want to add a TOC? It can be helpful so readers can quickly see what's coming.
```
Comment (cor3ntin) [inline src/D2662.tex:24]: I don't think that's necessary. pdfs readers will have an index and (hopefully) no one will print this

**Review by cor3ntin — COMMENTED** [review]
Thanks a lot! 
I left a few comments in case you want to reply directly, otherwise I'll merge later today and fix up the few issues you noticed.

File: src/D2662.tex (hunk 1 above)
Comment (lorihughes) [reply src/D2662.tex:131]: Okay, change this to "would simply work"

File: src/D2662.tex (hunk 2 above)
Comment (lorihughes) [reply src/D2662.tex:132]: Okay, so change that to user-friendliness

File: src/D2662.tex (hunk 3 above)
Comment (lorihughes) [reply src/D2662.tex:127]: If this is a phrase that your global audience will understand, then okay, but you do want to be sure that everyone can understand your content. That's the whole point of writing a paper!

File: src/D2662.tex (hunk 4 above)
Comment (lorihughes) [reply src/D2662.tex:111]: Your call.

File: src/D2662.tex (hunk 7 above)
Comment (lorihughes) [reply src/D2662.tex:24]: Some papers add a TOC, and some don't. Your call.

File: src/D2662.tex [hunk 8]
```diff
@@ -876,7 +890,7 @@ \section{Formal Wording}
 For a template argument \tcode{T}, if the \grammarterm{constant-expression} of a \grammarterm{pack-index-specifier} is type-dependent,
 \tcode{T...[\grammarterm{constant-expression}]} denotes a unique dependent type.
 Two such \grammarterm{pack-index-specifier}{s} refer to the same type only if their \grammarterm{constant-expression}{s} are
-equivalent\iref{temp.over.link} and their type are the same.
+equivalent \iref{temp.over.link} and their type are the same. %lah: what does the "their" refer to? The pack-index-specifiers or the constant-expressions? Also, we would need to alter this to "their types are the same" or "their type is the same".
```
Comment (cor3ntin) [inline src/D2662.tex:893]: the two \grammarterm{pack-index-specifier}{s}

File: src/D2662.tex [hunk 9]
```diff
@@ -19,152 +19,168 @@
...
 Historically in C++ (and C++-like languages), a single dot denotes member access.
-Reusing that syntax for pack indexing would introduce a semantics inconsistency.
+Reusing that syntax for pack indexing would introduce a semantics inconsistency and, more importantly, could close the door to future evolutions.
 
-More importantly, it may close the door to future evolutions.
-\paper{P1858R0} proposes \tcode{tuple-like.[N]} as sugar over \tcode{get<N>(tuple-like)}
-and \tcode{aggregate.[N]} returning the $Nth$ data member of an aggregate.
+\paper{P1858R0} proposes \tcode{tuple-like.[N]} as sugar over % lah: What do you mean "as sugar over"? If I'm not clear, then a global audience won't be clear. 
+\tcode{get<N>(tuple-like)}
+and \tcode{aggregate.[N]} returning the $Nth$ data member of an aggregate. % lah: WHAT is "returning the $Nth$ data member of an aggregate"---just the aggregate.[N] or BOTH get<N>(tuple-like) AND aggregate.[N]?
```
Comment (cor3ntin) [inline src/D2662.tex:129]: Just `aggregate.[N]`

Comment (lorihughes) [reply src/D2662.tex:129]: Then maybe this sentence should be "\paper{P1858R0} proposes \tcode{tuple-like.[N]} as syntactic sugar over \tcode{get<N>(tuple-like)} and \tcode{aggregate.[N]}, which leads to \tcode{aggregate.[N]} returning the $Nth$ data member of an aggregate."

File: src/D2662.tex (hunk 8 above)
Comment (lorihughes) [reply src/D2662.tex:893]: I'd add numbering to this to be excruciatingly clear: Two such \grammarterm{pack-index-specifier}{s} refer to the same type only if (1) their \grammarterm{constant-expression}{s} are
equivalent\iref{temp.over.link} and (2) their types are the same.

**Pull request merged**